tardis/montecarlo/montecarlo_numba/r_packet.py

```diff
@@ -30,12 +30,11 @@
     ENABLE_FULL_RELATIVITY,
 )
 
-
 class InteractionType(IntEnum):
-    BOUNDARY = 1
+    NO_INTERACTION = -1
+    BOUNDARY = 0
+    ESCATTERING = 1
```

Comment:
I'm a bit skeptical about this if changing these values leads to unintended side-effects (though tests pass) - can you double-check @andrewfullard (since you've recently worked with the numba montecarlo)?

Reply:
Yeah I personally don't see a need for this change to the enum values

Reply:
I agree with @andrewfullard.  Let's not change enum values if it's not necessary, especially since we don't have great tests for the sdec plot.

Reply:
It has been reverted ... a small note, like suggested above by @MarkMageeAstro, the following condition :
```
 mask_escatter = (
        self.data[packets_mode].packets_df["last_interaction_type"][
            self.packet_nu_range_mask
        ] == 1
```
becomes 
```
 mask_escatter = (
        self.data[packets_mode].packets_df["last_interaction_type"][
            self.packet_nu_range_mask
        ]
        == InteractionType.BOUNDARY
```
These changes are not that important as nothing is broken and associated with the enum values (operations or functionality) but it was a small point to make 🤔

Reply:
@marxwillia @andrewfullard The change "ESCATTERING = 1" **needs to be done** to give semantic name to `1` that is used throughout SDEC plot for comparison. I was just concerned if this change (which also requires changing BOUNDARY to some other value since it is currently 1) will cause any side-effects for rest of the codebase?

Reply:
Yes @jaladh-singhal, I have reverted it to that particular condition ... These values are very difficult to map as it has different values in different contexts.